examples/opentelemetry-web/examples/metrics/index.js

```diff
@@ -1,32 +1,33 @@
 const { DiagConsoleLogger, DiagLogLevel, diag } = require('@opentelemetry/api');
+const { metrics } = require('@opentelemetry/api-metrics');
 const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-http');
-const { MeterProvider } = require('@opentelemetry/sdk-metrics-base');
-const { Resource } = require('@opentelemetry/resources');
-const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+const { MeterProvider, PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics-base');
 
 // Optional and only needed to see the internal diagnostic logging (during development)
 diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG);
 
-const metricExporter = new OTLPMetricExporter();
-
 let interval;
 let meter;
 
-function stopMetrics() {
+async function stopMetrics() {
```

Comment:
async isn't really contributing much here I think. Awaiting the shutdown isn't actually doing anything here because the button click listener doesn't await this.

Reply:
`await`ing still delays `metrics.disable()`, right? :thinking: 
I removed the `async` and chained `metrics.disable()` to the shutdown promise instead of `await`ing it in [118c38d](https://github.com/open-telemetry/opentelemetry-js/pull/3145/commits/118c38d7fd657d66a61da20b5fd7011a9951c168). :slightly_smiling_face:

Reply:
> awaiting still delays metrics.disable(), right? 🤔

yes, but the order here actually doesn't matter anyway

> I removed the async and chained metrics.disable() to the shutdown promise instead of awaiting it in [118c38d](https://github.com/open-telemetry/opentelemetry-js/pull/3145/commits/118c38d7fd657d66a61da20b5fd7011a9951c168). 🙂

ok